editor/gui/editor_toaster.cpp

```diff
@@ -371,13 +371,20 @@ Control *EditorToaster::popup(Control *p_control, Severity p_severity, double p_
 	p_control->set_h_size_flags(SIZE_EXPAND_FILL);
 	hbox_container->add_child(p_control);
 
-	// Close button.
+	// Add buttons.
 	if (p_time > 0.0) {
+		Button *copy_button = memnew(Button);
+		copy_button->set_flat(true);
+		copy_button->set_focus_mode(FOCUS_NONE);
```

Comment:
That's inconsistent with the other button 🤔

Reply:
Changed, thought it looked a bit weird after copying with the focus outline, but it's probably better for consistency's sake.